src/filters/2d_backend.class.ts

```diff
@@ -1,78 +1,72 @@
-//@ts-nocheck
-(function (global) {
-  var fabric = global.fabric,
-    noop = function () {};
+/**
+ * Canvas 2D filter backend.
+ */
+import { noop } from '../constants';
+import { T2DPipelineState } from './typedefs';
 
-  fabric.Canvas2dFilterBackend = Canvas2dFilterBackend;
+export class Canvas2dFilterBackend {
+  evictCachesForKey = noop;
+  dispose = noop;
+  clearWebGLCaches = noop;
 
-  /**
-   * Canvas 2D filter backend.
-   */
-  function Canvas2dFilterBackend() {}
-
-  Canvas2dFilterBackend.prototype =
-    /** @lends fabric.Canvas2dFilterBackend.prototype */ {
-      evictCachesForKey: noop,
-      dispose: noop,
-      clearWebGLCaches: noop,
+  constructor() {
+    return this;
```

Comment:
??

Reply:
i don't know... let me remove it, i don't remember